assignment4/assignment4.js

```diff
@@ -0,0 +1,36 @@
+var fs = require('fs');
+var data1, data2;
+fs.readFile('file1.txt', function (err, data) {
+  if (err) {
+      return console.error(err);
+  }
+  data1=data.toString();
+    fs.appendFile('file2.txt', data1 , function(error) {
+      if (error) {
+        console.log('Error:- ' + error);
+        throw error;
+      }
+      fs.readFile('file2.txt', function (err, data) {
+        if (err) {
+          return console.error(err);
+        }
+        data2=data.toString();
+        fs.writeFile('file3.txt', '', function (err) {
```

Comment:
Why are you creating 3 functions??????????????????????????????????
One for writeFile and then to append to the same file and then reading from same file. Is there any logic that I am missing.